_data/data.yaml

```diff
@@ -1458,7 +1458,7 @@ tabs:
             source: 'http://www.nbcnews.com/politics/2016-election/full-list-donald-trump-s-rapidly-changing-policy-positions-n547801'
           -
             text: 'Defund Planned Parenthood.'
-            status: notStarted
+            status: Started
```

Comment:
http://www.politifact.com/truth-o-meter/promises/trumpometer/promise/1357/defund-planned-parenthood/

It's in the works for this Sunday.